tools/requirements.map

```diff
@@ -0,0 +1,19 @@
+# Map of import names to packages used by all ./b commands
```

Comment:
We probably don't need all these dependencies. Will need to go through the tools and find minimum set of dependencies, like I did with the 4Tel tools. Perhaps we should wait for #364 first.

Reply:
This is why I assigned it to you :P

Reply:
Did you end up going through these?

Reply:
Yeah